cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/namespace/DefaultNamespaceEnsurer.java

```diff
@@ -52,7 +53,11 @@ protected void doStart() {
               // if there is no exception, assume successfully created and break
               LOG.info("Successfully created namespace '{}'.", NamespaceMeta.DEFAULT);
               notifyStarted();
-            } catch (AlreadyExistsException e) {
+            } catch (FileAlreadyExistsException e) {
+              LOG.warn("Got exception while trying to create namespace '{}'.", NamespaceMeta.DEFAULT, e);
```

Comment:
This means the namespace already exists, right? This message seems to be misleading in that case. Would be better if we just info log `LOG.info("Default namespace already exists.");`